cdap-ui/app/features/hydrator/controllers/list-ctrl.js

```diff
@@ -171,19 +171,19 @@ angular.module(PKG.name + '.feature.hydrator')
     }
 
     function fetchDrafts() {
-      mySettings.get('adapterDrafts')
+      mySettings.get('hydratorPipelines')
         .then(function(res) {
           let draftsList = myHelpers.objectQuery(res, $stateParams.namespace);
           if (!angular.isObject(draftsList)) {
             return;
           }
           if (Object.keys(draftsList).length) {
-            angular.forEach(res[$stateParams.namespace], function(value, key) {
+            angular.forEach(res[$stateParams.namespace], function(value) {
               vm.statusCount.draft++;
               vm.pipelineList.push({
                 isDraft: true,
-                name: key,
-                id: key,
+                name: value.name,
+                id: value.__ui__.draftId,
```

Comment:
For the non-migrated drafts it won't have draftId. But all new drafts will have draftId.